tests/suites/test_suite_md.function

```diff
@@ -271,9 +271,9 @@ exit:
 /* END_CASE */
 
 /* BEGIN_CASE */
-void mbedtls_md_hmac( char * text_md_name, int trunc_size,
-                      data_t * key_str, data_t * src_str,
-                      data_t * hex_hash_string )
+void mbedtls_md_hmac( char *text_md_name,
+                      data_t *key_str, data_t *src_str,
```

Comment:
What happened to the `int trunc_size` arg? It's used later in the function but declaration was removed...

Reply:
Removing the argument seems OK but it was done in the wrong commit, please fix.